upstream.wrap

```diff
@@ -0,0 +1,10 @@
+[wrap-file]
+directory = tclap-1.2.1
+source_url = https://sourceforge.net/projects/tclap/files/tclap-1.2.1.tar.gz
+source_filename = tclap-1.2.1.tar.gz
+source_hash = 9f9f0fe3719e8a89d79b6ca30cf2d16620fba3db5b9610f9b51dd2cd033deebb
+
+#TODO: Add patch when available in Meson's WrapDB.
```

Comment:
You don't need to have this here. It is in fact counterproductive. Wrap tooling will add these automatically.

Reply:
Do you mean only the comment? Or the ``source_filename`` and ``source_hash`` too?

Reply:
The comment.

Reply:
Just amended&pushed my commit because I forgot to include a cpp file. (Sorry ...)
Removed the comment as suggested.